src/Spectre.Algorithms.Tests/Results/DivikResultTests.cs

```diff
@@ -52,7 +54,25 @@ public void SetUpFixture()
         [Test]
         public void Save()
         {
-            Assert.Throws<NotImplementedException>(() => _result.Save("test-path.json"));
+            string path = TestContext.CurrentContext.TestDirectory + "\\..\\..\\..\\test-path.json";
+            _result.Save(path);
+
+            Assert.True(File.Exists(path), "File doesn't exist");
+            Assert.AreNotEqual(0, new FileInfo(path).Length, "File is empty");
+
+            string jsonData = File.ReadAllText(path);
+            DivikResult deserialisedResult = JsonConvert.DeserializeObject<DivikResult>(jsonData);
+
+            Assert.AreEqual(_result.AmplitudeFilter, deserialisedResult.AmplitudeFilter, "The AmplitudeFilter differs from original");
+            Assert.AreEqual(_result.AmplitudeThreshold, deserialisedResult.AmplitudeThreshold, "The AmplitudeThreshold differs from original");
+            Assert.AreEqual(_result.Centroids, deserialisedResult.Centroids, "The Centroids differ from original");
+            Assert.AreEqual(_result.Merged, deserialisedResult.Merged, "The Merged differs from original");
+            Assert.AreEqual(_result.Partition, deserialisedResult.Partition, "The Partition differs from original");
+            Assert.AreEqual(_result.QualityIndex, deserialisedResult.QualityIndex, "The QualityIndex differs from original");
+            Assert.AreEqual(_result.VarianceFilter, deserialisedResult.VarianceFilter, "The VarianceFilter differs from original");
+            Assert.AreEqual(_result.VarianceThreshold, deserialisedResult.VarianceThreshold, "The VarianceThreshold differs from original");
+            Assert.AreEqual(_result.Subregions, deserialisedResult.Subregions, "The Subregions differ from original");
+            File.Delete(path);
```

Comment:
Shouldn't this part be removed in `finally` clause? Or do we want to preserve any information (i.e. in the form of created JSON file)?